cmd/vic-machine/create/create.go

```diff
@@ -702,6 +703,10 @@ func (c *Create) Run(clic *cli.Context) (err error) {
 		vchConfig.StorageQuota = quotaBytes
 	}
 
+	if c.ContainerCount != nil && *c.ContainerCount > 0 {
```

Comment:
Why this code differs than that in `configure` part ? Needn't set `vchConfig.ContainerCount = 0`?

Reply:
The default value during creation is 0.